internal/docs/doc-login.md

```diff
@@ -35,13 +35,34 @@ If you don't provide an alias, the server name is used as an alias, so you shoul
 add an alias.
 
 ```
-mim login add --server="https://my.datahub.server" --token="<valid token>" --alias="server1"
+mim login add --type=client --server="https://my.datahub.server" --token="<valid token>" --alias="server1"
 ```
 
 You cannot change the alias on an existing profile, you need to delete it first.
 
-Since a token has a limited validity, we suggest you use a setup with a clientId and a clientSecret
-instead.
+There are 3 types of logins that are supported: token, client, user.
+
+ * token - you must provide a valid token yourself - this is the worst option
+ * client - this is a combination if a client id and secret - second worst option
```

Comment:
I dont think we should use opinionated language. e.g. in the case the cli is running as part of a batch job this is the best and only viable option.